ui/lib/core/addon/components/form-field.hbs

```diff
@@ -305,7 +305,7 @@
           {{on "input" this.onChangeWithEvent}}
           {{on "change" this.onChangeWithEvent}}
           {{on "keyup" this.handleKeyUp}}
-          class="input {{if this.validationError 'has-error-border'}} has-bottom-margin-m"
+          class="input {{if this.validationError 'has-error-border' 'has-bottom-margin-m'}}"
```

Comment:
I noticed this while testing the form validation and when there is an error the added margin pushes the message down which doesn't align with how the message is rendered for other form types like the select element for example.